DSA-Campaign-Uplift-Estimation/src/main/webapp/About/about.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About</title>
+    <link rel="stylesheet" href="../Resources/NavBar/navbar_style.css"></link>
+    <script src="../Resources/NavBar/navbar_script.js"></script>
+    <link rel="stylesheet" href="style-about.css"></link>
+  </head>
+  <body onload="getLogsStatus();">
+    <div id="mySidenav" class="sidenav">
+      <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+      <img src="../Resources/images/DSA-campaign-white.png" style="max-width: 50%; max-height: 50%; margin-left: auto; margin-right: auto; display: block;"/>
+      <a href="/Home/home.html">Home</a>
+      <a href="/Create/create.html">Create</a>
+      <a href="/Compare/compare.html">Compare</a>
+      <a href="/About/about.html">About</a>
+      <a id="loginout-link" href="#">Logout</a>
+    </div>
+
+    <header>
+      <div class="header">
+        <span style="font-size: 30px; cursor: pointer; float: left; color: #f2f2f2;" onclick="openNav()">&#9776;</span>
+        <a href="#default" class="title">About</a>
+      </div>
+    </header>
+	
+    <div class="content">
+        <h3>Project Overview</h3>
+        <p>This project aims to address two main problems with the current Dynamic Search Ads framework. The first
+            involves the lack of transparency associated with automated products and the limited predictability of DSA and 
+            machine learning in general. Users have no guarantee that using DSA will produce the results they intend,
+            creating some risk for users attempting to transition over from their traditional keyword campaigns
+            to DSA.</p>
+        <p>The second issue involves the sometimes unnecessary complexity of DSA. Currently over 40% of created DSA
+            campaigns have a serving rate of 0, meaning that the campaign generates no clicks and therefore brings 
+            no revenue. This is due to some misconfiguration while setting up the DSA campaigns, and since this happens 
```

Comment:
I would leave this metric out of the about page, as these are internal metrics that we don't want to share outside.

Reply:
done